mlrun/serving/remote.py

```diff
@@ -454,3 +459,67 @@ def _get_data(self, data, headers):
         ) and isinstance(data, (str, bytes)):
             data = json.loads(data)
         return data
+
+
+class MLRunAPIRemoteStep(RemoteStep):
+    def __init__(
+        self, method: str, path: str, fill_placeholders: Optional[bool] = None, **kwargs
+    ):
+        """
+        Graph step implementation for calling MLRun API endpoints
+
+        :param method:  HTTP method (GET, POST, ...) as default defines the method of the request, if empty string or
+                        None the method will be taken from the event (default to POST)
```

Comment:
There are 3 different defaults mentioned here. I don't think a user would be able to make sense of it, so it needs a bit of a rewrite.